src/index.html

```diff
@@ -11,7 +11,177 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="form-field block">
+        <legend>Personal information</legend>
+        <div class="input">
+          <label for="aaa">Surname:</label>
+          <input
+            type="text"
+            name="attribute"
+            autocomplete="off"
+            required
+          >
```

Comment:
It would be best if you wrapped inputs into labels, so user can activate input by clicking on the whole row
Also, what's the purpose of "for" attribute of the label?
Please fix all cases
![image](https://user-images.githubusercontent.com/96409563/222509307-9fd0df78-42f5-407b-b79e-202911dd1b0e.png)